app.test.js

```diff
@@ -0,0 +1,24 @@
+
+import { describe, expect, it } from 'vitest'
+// import app from '/server.js';
+// const { default: test } = require('node:test');
+import { app } from './app'
+
+describe('Get health route', () => {
+  it('checks that status is 200', async () => {
+    const res = await app.inject({
+      method: 'GET',
+      url: '/health'
+    })
+
+    expect(res.statusCode).toBe(200)
+    expect(res.body).toEqual('ok')
+  })
+  it('checks fot 404 status when there are errors', async () => {
```

Comment:
Be more verbose about the test. `"when there are errors"` is too general

Reply:
done